contents/handbook/growth/marketing/index.md

```diff
@@ -1,59 +1,97 @@
 ---
-title: Overview
+title: Manifesto
 sidebar: Handbook
 showTitle: true
 ---
 
-The Marketing team's main focus is acquisition. Our specific objective is to _get as many relevant developers and technical product managers as possible to visit posthog.com_ 
-
-Once a visitor lands on our website, the [Growth](/handbook/people/team-structure/growth) team are responsible for maximizing activation. 
-
-## Positioning
-
-We use a positioning statement to help us think about how we can continue to improve and evolve PostHog ahead of alternative tools:
-
-_For product managers who need to understand their users, PostHog is the only open-source, all-in-one analytics platform that offers everything from funnel analysis and cohort tracking, to feature flags and session recording._ 
-
-_Unlike traditional tools, PostHog offers self-hosted deployments so that data doesn't need to be shared with anyone -- not even PostHog._ 
-
-See [our messaging framework for more information](/handbook/growth/marketing/messaging_framework) about positioning and how we bring it to life. 
-
-### Value propositions
-
-We have used our messaging framework to then articulate a series of [value propositions](/handbook/growth/marketing/value-propositions) ('value props'). These communicate PostHog's benefits in a clear and specific way to our target user, and help us ensure that we are communicating consistently across all of our different channels. This is particular important for paid ads. 
-
-### Target user
-
-We have built out two user personas and will continue to flesh these out and add more as we gather more information:
-
-- Sam Software, a senior software engineer
-- Pam Product, a senior product manager
-
-See our [customer personas page](/handbook/growth/marketing/customer-personas) for more information about what is important to these users and how we ensuring they have the best possible experience with PostHog across all product and marketing touchpoints. 
-
-### Products
-
-An **open source product analytics platform**, PostHog addresses the lack of choice and control amongst disconnected analytics solutions by offering a **unified platform** with **control** over hosting, pricing, source, data, privacy and security.
-
-- **PostHog Open Source** is positioned to service 1-3 users in teams of any size, from hobbyists to enterprise deployments.
-- **PostHog Scale** is positioned to service collaborative teams or enterprises who need to retain control of their data. 
-- **PostHog Enterprise** is positioned to service large enterprises who have more complex or involved infra and config needs. 
-- **PostHog Cloud** is positioned to service collaborative teams or enterprises who value simplicity or don't need to retain control of their data.
+## Everything starts with our values
```

Comment:
I am (very) happy with these as things to care about, but a little confusing to have company values and marketing values.

Reply:
I'll rename (principles or something similar) to avoid that confusion